CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 - No longer transitively depend on `time` 0.1  [EventStoreDB-Client-Rust#160](https://github.com/EventStore/EventStoreDB-Client-Rust/pull/160)
+- Update to tonic 0.9  [EventStoreDB-Client-Rust#162](https://github.com/EventStore/EventStoreDB-Client-Rust/pull/162)
```

Comment:
You don't need to update this file, it will be done automatically by one of our bots.